includes/admin/class-coblocks-getting-started-page.php

```diff
@@ -59,7 +60,7 @@ public function screen_page() {
 			apply_filters( 'coblocks_getting_started_screen_capability', 'manage_options' ),
 			'coblocks-getting-started',
 			array( $this, 'content' ),
-			'data:image/svg+xml;base64,' . base64_encode( $svg )
```

Comment:
After inspecting further, the issue isn't that we're missing the `$position` parameter (because it is optional and defaults to `null`), it's because we were passing the base64 encoded svg icon as the `$position` parameter. The passed parameters were not updated when switching this to [`add_management_page`](https://developer.wordpress.org/reference/functions/add_management_page/) from [`add_menu_page`](https://developer.wordpress.org/reference/functions/add_menu_page/).

Reply:
I've pushed an update which removes the `$position` parameter from our calls since we only needed to remove the svg icon.